tests/model_serving/model_runtime/mlserver/gpu/test_mlserver_gpu_rbac.py

```
@@ -0,0 +1,103 @@
+        When authorized, unauthenticated, and invalid token requests are sent
+        Then authorized gets 200, others get 401 or 403.
+        """
+        base_url = get_exposed_isvc_url(isvc=mlserver_inference_service)
+        model_name = mlserver_inference_service.name
+        infer_url = f"{base_url}/v2/models/{model_name}/infer"
+
+        authorized_response = requests.post(
+            url=infer_url,
+            json=ONNX_RESNET50_REST_INPUT_QUERY,
+            headers={"Authorization": f"Bearer {authorized_inference_token}"},
+            verify=False,
```

Ruff (0.16.0)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)